src/comppackdb.cpp

```diff
@@ -114,7 +114,9 @@ void CompPackDatabase::parse_entries(std::string& db_data)
             auto err = sqlite3_exec(
                     _sqliteDb.get(), "ROLLBACK", nullptr, nullptr, &errmsg);
             if (err != SQLITE_OK)
+            {
                 LOG("sqlite error: %s", errmsg);
+            }
```

Comment:
I prefer using the do {} while (0) trick in the LOG macro instead of changing all the call sites

Reply:
I rarely use macros, so wasn't aware of the trick.  Only two call sites though so either solution is a good one imo :D